docs/bpmn-support.adoc

```diff
@@ -63,11 +63,11 @@ The default rendering uses `white` as fill color and `black` as stroke color.
 |icon:flask[]
 |Icon will be subject to a large change +
 *icon*: the task icon is currently derived from the https://github.com/primer/octicons/blob/638c6683c96ec4b357576c7897be8f19c933c052/icons/person.svg[primer octicons person svg]
-|===
 
 |Call Activity
 |
 |
+|===
```

Comment:
:+1:

Reply:
:information_source: I have seen the same kind of change in #271, so merge conflicts will occur :smiley_cat: